components/counter/Counter.js

```diff
@@ -6,25 +6,29 @@ import theme from './theme.css';
 
 class Counter extends PureComponent {
   static propTypes = {
+    borderColor: PropTypes.oneOf([ 'neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby', 'tealdark' ]),
```

Comment:
Can't we use a `border` prop that is either `true` or `false`? I guess we're never going to use a gold counter with a ruby border?

Reply:
ignore my comment, i'm wrong 👍